stl/inc/flat_set

```diff
@@ -430,27 +447,25 @@ public:
     }
 
 private:
-    void _Assert_after_sorted_input() const {
-        _STL_ASSERT(_STD is_sorted(cbegin(), cend(), _Get_comp_v()), "Input was not sorted!");
-        if constexpr (!_Multi) {
-            _STL_ASSERT(_Is_unique(), "Input was sorted but not unique!");
-        }
-    }
-
-    bool _Is_unique() const {
-        if (empty()) {
-            return true;
-        }
-        const const_iterator _End = cend();
-        const_iterator _It        = cbegin();
-        while (++_It != _End) {
-            if (_Keys_equal(*(_It - 1), *_It)) {
-                return false;
+    bool _Check_sorted(const_iterator _It, const const_iterator _End) const {

```

Comment:
Suggestion: Due to the naming, mistaken usage is possible. This should be `nodiscard`.

Similarly for `_Check_where` below.